mxd/docs/File Transfer Azure to Azure.md

```diff
@@ -0,0 +1,416 @@
+# File Transfer: Azure Storage to Azure Storage
+
+## 1. Description
+This tutorial illustrates the process through which a provider participant can transfer a file stored in its Azure Storage to another Azure Storage belonging to a consumer participant.  
+We assume `Alice` as a provider participant and `Bob` as consumer participant.
+
+## 2. Existing Resources
+
+Following resources have already been configured.
+
+| Resource                        | Alice                   | Bob                   | Description                                                                                                                                                                                                                                                                                                                                    |
+|---------------------------------|-------------------------|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Azure Account Name              | aliceazureaccount       | bobazureaccount       | Configured Azure Account Name                                                                                                                                                                                                                                                                                                                  |
+| Azure Account Key               | YWxpY2VhenVyZWtleQ==    | Ym9iYXp1cmVrZXk=      | Configured Azure Account key                                                                                                                                                                                                                                                                                                                   |
+| Azure Account Key Name in Vault | aliceazureaccount-key   | bobazureaccount-key   | Key Name against which Account Key has been stored in the participant's vault. This key name will be used while creating asset. It has been done to avoid leaking secret keys.                                                                                                                                                                 |
```

Comment:
better: "The alias under which the Account Key has been stored in the participant's vault"